### PR TITLE
Qobo-WP-Custom-Admin-Theme update

### PR DESCRIPTION
**Pull request by flangofas**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -105,7 +105,7 @@
         "qobo/qobo-wp-custom-theme-path": "1.0.1",
         "qobo/qobo-wp-brand": "1.0.0",
         "qobo/qobo-wp-shortcodes": "1.0.0",
-        "qobo/qobo-wp-custom-admin-theme": "1.0.0",
+        "qobo/qobo-wp-custom-admin-theme": "1.0.1",
```
Comment (mamchenkov) [inline composer.json]: @flangofas hmm ... why does it require itself? :)

**Comment by flangofas** [comment]
@mamchenkov it fixes some styles of WP admin panel where input or select fields overlap in Widget box  containers(parent div).

File: composer.json (hunk 1 above)
Comment (flangofas) [inline composer.json]: @mamchenkov it fixes some styles of WP admin panel where input or select fields overlap in Widget box containers(parent div).

Comment (flangofas) [inline composer.json]: @mamchenkov :+1:

**Pull request merged**